chapters/002-smol-internet/README.md

```diff
@@ -196,30 +154,22 @@ default via 10.1.2.1 dev eth0
 10.1.2.0/24 dev eth0 proto kernel scope link src 10.1.2.2
 ```
 
-But what is actually happening in that request/reply? For a detailed explanation,
-checkout the [appendix on ip v. mac addresses](../appendix/ip-and-mac-addresses.md).
+But what is actually happening in that request/reply? For a detailed explanation, checkout the [appendix on ip v. mac addresses](../appendix/ip-and-mac-addresses.md).
 
 Now back to our regularly scheduled exploration!
 
 ## Make those networks communicate with each other!
 
-How do machines communicate across networks? Well, first they need to have a
-router. Sure, docker has its own built in router, but we want to build our own.
-What is a router, but just another machine on the network. The router just has 2
-special properties:
+How do machines communicate across networks? Well, first they need to have a router. Sure, docker has its own built in router, but we want to build our own.  What is a router, but just another machine on the network. The router just has 2 special properties:
```

Comment:
```suggestion
How do machines communicate across networks? Well, first they need to have a router. Sure, docker has its own built in router, but we want to build our own.  What is a router, but just another machine on the network. A router just has 2 special properties that make it a router instead of just another machine on the network:
```